### DISCUSSION TITLE
Cannot able to connect to postgres.

### DISCUSSION THREAD

**Discussion by a-imran99** [Ylianst/MeshCentral#4639] [Q&A]
Hi @Ylianst @si458 , I have been trying to connect Postgres with Mesh-central and gave the below settings in config.json 

"postgres": 
{
"user": "postgres",
"password": "pass123",
"port": "5432",
"host": "localhost",
}

when this is done, pg@8.7.1 is being installed by mesh-central, But the below error throws, we have a nginx-ingress as reverse proxy

The error,

Installing pg@8.7.1...
ERROR: Unable to install required module "pg@8.7.1". MeshCentral may not have access to npm, or npm may not have suffisent rights to load the new module. To manualy install this module try:

   cd "/opt/meshcentral/meshcentral"
   npm install pg@8.7.1
   node node_modules/meshcentral

It would be great if you guys make a comment.

**Comment by dinger1986** [comment]
whats your host machine?

**Reply by a-imran99** [reply]
I am using docker container. @dinger1986

**Reply by dinger1986** [reply]
ah ok, did you try and manually install pg@8.7.1? I assume thats whats wrong here and its permissions.

I dont do alot with docker so dont know much more

**Reply by a-imran99** [reply]
yes now I have commented on code, where pgtools and PG is automatically installed if postgres details are given. and added those things to package.json instead. @dinger1986

**Reply by dinger1986** [reply]
ok cool, so working?

**Reply by a-imran99** [reply]
Just waiting for the container build to complete on development server, meanwhile trying to replicate in local. :) @dinger1986
